Sources/Realtime/RealtimeClient.swift

```diff
@@ -594,9 +611,10 @@ public class RealtimeClient: TransportDelegate {
   /// - parameter topic: Topic of the channel
   /// - parameter params: Optional. Parameters for the channel
   /// - return: A new channel
+  @available(*, deprecated, renamed: "channel(_:options:)")
```

Comment:
should we really deprecate this?

aren't there other params that can be passed other than `ChannelOption`? Having this raw `[String: Any]` I think is still valid.

I'm not sure about this because I'm not used to the realtime feature, what do you think @foodisbeast ?

Reply:
Unless this package is meant to support Phoenix realtime generally, supabase realtime or `realtime-js` library only has these options available for initializing a channel (See [RealtimeChannelOptions](https://github.com/supabase/realtime-js/blob/c9dc9c953cb6bb3e9440749d393e944b63a0a0f9/src/RealtimeChannel.ts#L15-L27) from `realtime-js`), as well as their [documention](https://supabase.com/docs/guides/realtime) for realtime only makes mention/use of RealtimeChannelOptions once when talking about [Presence Key](https://supabase.com/docs/guides/realtime/presence)
```typescript
import { createClient } from '@supabase/supabase-js'

const channelC = supabase.channel('test', {
  config: {
    presence: {
      key: 'userId-123',
    },
  },
})
```